frontend/src/routes/Infrastructure/Clusters/ClusterPools/ClusterPools.tsx

```diff
@@ -290,11 +291,7 @@ export function ClusterPoolsTable(props: {
             header: t('table.cluster.statuses'),
             cell: (clusterPool: ClusterPool) => {
               if (isClusterPoolDeleting(clusterPool)) {
-                return (
-                  <Label color="grey" icon={<TrashIcon />}>
-                    {t('Deleting')}
-                  </Label>
-                )
+                return <AcmInlineStatus type={StatusType.progress} status={t('Destroying')} />
```

Comment:
```suggestion
                return <AcmInlineStatus type={StatusType.progress} status={t('destroying')} />
```

For some reason the translation key starts with lower case.